askup/admin.py

```diff
@@ -124,7 +124,7 @@ def get_inline_instances(self, request, obj=None):
     @staticmethod
     def domains(obj):
         """Return email-patters of the organization comma separated."""
-        return ", ".join(tuple(domain.name for domain in obj.domain_set.all()))
+        return ", ".join((domain.name for domain in obj.domain_set.all()))
```

Comment:
odd parentheses `.join(domain.name for domain in obj.domain_set.all())` is fair enough.